pkg/chunk/gcp/bigtable_index_client.go

```diff
@@ -247,50 +259,6 @@ func (s *storageClientColumnKey) QueryPages(ctx context.Context, queries []chunk
 	return lastErr
 }
 
-func (s *storageClientColumnKey) query(ctx context.Context, query chunk.IndexQuery, callback func(result chunk.ReadBatch) (shouldContinue bool)) error {
```

Comment:
How does removal of this `query()` function relate to the rest of the commit?

Reply:
Doesn't. But that codepath is not being called. Probably should have been another PR.

Reply:
As a separate commit it might be acceptable, but surely having all changes in one commit consistent and described by the commit message is a very basic standard to set?